packages/admin-next/dashboard/src/routes/draft-orders/draft-order-detail/components/draft-order-summary-section/draft-order-summary-section.tsx

```diff
@@ -129,9 +193,15 @@ const ItemBreakdown = ({ draftOrder }: { draftOrder: DraftOrder }) => {
     throw error
   }
 
+  const variantBasedItems = draftOrder.cart.items.filter(
+    (i) => i.variant_id !== null
+  )
+
+  const customItems = draftOrder.cart.items.filter((i) => i.variant_id === null)
```

Comment:
Custom items are not visible

![Screenshot 2024-03-20 at 09 12 08](https://github.com/medusajs/medusa/assets/16856471/d6b82cb9-fdd3-4240-a42f-103c45640e3e)

Reply:
Hmm thats weird they show up on my end. Do you see any errors in your console, or could you maybe share the JSON for your custom item so I can try making copy of it

Reply:
It's fixed in the latest commit - all good 👍